### PR TITLE
fix flake8

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Travis is still red

**Comment by pedrobaeza** [comment]
No luck :(

**Comment by elicoidal** [comment]
@gurneyalex Shall we finish this one or close it?

File: project_members/__manifest__.py [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
- 'license': 'AGPL-3',
- 'authors': 'Alexandre Fayolle <alexandre.fayolle@camptocamp.com>, Odoo Community Association (OCA)',
+# -*- coding: utf-8 -*-
+# Copyright (C) Odoo Community Association (OCA)
+# @author: Alexandre Fayolle
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'project members',
+    'description': 'add members to projects',
+    'summary': 'add members to projects'
```
Comment (MiquelRForgeFlow) [inline project_members/__manifest__.py]: should end with a comma

**Comment by elicoidal** [comment]
I have resolved conflicts + add comma.
Let's wait for Travis

File: project_members/__manifest__.py [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+{
+    'name': 'Project members',
+    'description': 'Add members to projects',
+    'summary': 'Add members to projects',
+    'version': '10.0.1.0.0',
+    'category': 'Project',
+    'depends': ['project'],
+    'data': ['views/project.xml'],
+    'license': 'AGPL-3',
+    'authors': ('Alexandre Fayolle <alexandre.fayolle@camptocamp.com>,'
```
Comment (MiquelRForgeFlow) [inline project_members/__manifest__.py]: no need of ()

File: oca_custom/__manifest__.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 {
     'name': 'OCA Custom Settings',
     'summary': 'Custom Settings for OCA Instance',
-    'version': '10.0.0.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Custom',
     'author': [
```
Comment (MiquelRForgeFlow) [inline oca_custom/__manifest__.py]: shouldn't be a list. should be a string like in other modules

File: oca_custom/__manifest__.py [hunk 4]
```diff
@@ -5,7 +5,7 @@
 {
     'name': 'OCA Custom Settings',
     'summary': 'Custom Settings for OCA Instance',
-    'version': '10.0.0.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Custom',
```
Comment (mreficent) [inline oca_custom/__manifest__.py]: Missing required key "license"

File: project_members/__manifest__.py [hunk 5]
```diff
@@ -1,8 +1,16 @@
...
- 'data': ['views/project.xml'],
- 'license': 'AGPL-3',
- 'authors': 'Alexandre Fayolle <alexandre.fayolle@camptocamp.com>, Odoo Community Association (OCA)',
+# -*- coding: utf-8 -*-
+# Copyright (C) Odoo Community Association (OCA)
+# @author: Alexandre Fayolle
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project members',
+    'description': 'Add members to projects',
```
Comment (MiquelRForgeFlow) [inline project_members/__manifest__.py]: remove description, it's deprecated

File: project_members/models/project.py [hunk 6]
```diff
@@ -4,5 +4,10 @@
 class ProjectProject(models.Model):
```
Comment (mreficent) [inline project_members/models/project.py]: add -*- coding: utf-8 -*-` in this file

**Comment by elicoidal** [comment]
I will fix it tomorrow

**Comment by MiquelRForgeFlow** [comment]
flake8 errors...

**Comment by elicoidal** [comment]
YEah I will continue fixing...

**Comment by elicoidal** [comment]
@mreficent It looks like pylint is over: I do not know how to solve the new error on Odoo.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by elicoidal** [comment]
@pedrobaeza shall we merge?

**Comment by pedrobaeza** [comment]
Well, this still continues red, but I suppose the situation is better. In upper branches is green any way.

**Pull request merged**